mirdata/download_utils.py

```diff
@@ -220,10 +220,7 @@ def untar(tar_path, cleanup=False):
     cleanup: bool, default=False
         If True, remove tarfile after untarring.
     """
-    if tar_path.endswith('tar.gz'):
-        tfile = tarfile.open(tar_path, 'r:gz')
-    else:
-        tfile = tarfile.TarFile(tar_path, 'r')
+    tfile = tarfile.open(tar_path, 'r')
```

Comment:
Do you know why the original code was necessary/why it's not necessary now?

Reply:
Not completely, I was using that code to open `tar.gz` files but it breaks with `Beatles` so I changed back to the simple code.